config/reports.json

```diff
@@ -80,18 +90,20 @@
 		"fmp": {
 			"name": "First Meaningful Paint",
```

Comment:
As an aside, we may want to remove this.. We're moving away from FMP (for now, at least) in LH. Instead show FCP to align with CrUX and RUM apis.

Reply:
Done.

Writing down some notes about the switchover here before I forget:

- FCP is not available in Lighthouse, but WPT does expose it in the HAR as `_chromeUserTiming.firstContentfulPaint`
- renamed and rewrote fmp.sql histogram/timeseries queries to use above FCP metric
- renamed the metric in the report.json config and updated its description
- went through the timeseries data to find the first complete crawl with FCP data (2016_12_15) and set the minDate in the metric config
- script to backfill FCP timeseries data: `sql/generateReport.sh -d "fcp.json"`
- script to backfill FCP histogram data: `sql/getBigQueryDates.sh pages '' 2016_12_15 | xargs -I date sql/generateReport.sh -d date/fcp.json`